priv/mssql2012.sql

```diff
@@ -200,6 +201,7 @@ GO
 SET ANSI_PADDING ON
 GO
 CREATE TABLE [dbo].[privacy_list](
+	[server] [nvarchar](250) NOT NULL,
```

Comment:
Note: for other DBs we have a primary key here. Maybe it would be good to add it for MS SQL as well for consistency.